.travis.yml

```diff
@@ -1,19 +1,34 @@
+
 language: python
+sudo: true
+group: beta
+dist: xenial
 python:
     - 2.7
     - 3.5
-script:
-    - ./setup.py test
-    - pip uninstall -y greenstack && ./setup.py develop && python -c "import v8py"
-
-dist: trusty
-sudo: false
-group: beta
+    - 3.6
+    - 3.7
 
 install:
-    - pip install twine pytest greenstack
+    - sudo apt update && sudo apt install patchelf
+    - pip install --upgrade pip && pip install twine pytest
+    # greenstack doesn't build for python 3.7 and above
```

Comment:
I'd be willing to abandon greenstack support, I seriously doubt anyone is using it at all. That can be a new PR though.

Reply:
In the interim would this workaround be okay? I agree that removing greenstack should be a different PR.

Reply:
Sure